src/app/components/shell/footer/footer.js

```diff
@@ -32,6 +32,9 @@ function Footer({data}) {
                         <a href="/tos">Terms of Use</a>
                         <a href="/license">Licensing</a>
                         <a href="/privacy-policy">Privacy Policy</a>
+                        <a href="https://app.zenhub.com/workspaces/openstax-bit-5bacfe3f4b5806bc2bea3764/issues/openstax/business-intel/1423">
```

Comment:
@RoyEJohnson Was this the URL you meant to use?

Reply:
Yes, this was given to me by Allison.

Reply:
I pushed an amended commit with the updated URL.